Win/Mineways.vcxproj

```diff
@@ -76,6 +76,8 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
     <LinkIncremental>false</LinkIncremental>
     <TargetName>$(ProjectName)32</TargetName>
+    <IncludePath>$(VC_IncludePath);$(WindowsSDK_IncludePath);./curl/include;</IncludePath>
+    <LibraryPath>$(VC_LibraryPath_x86);$(WindowsSDK_LibraryPath_x86);./curl/lib;</LibraryPath>
```

Comment:
Please do add this path to all projects, debug/release and win32/x64. Right now the x64 versions don't link (I guess they need the 64 bit version of curl.lib?).